src/validation/coinbase.cairo

```diff
@@ -70,42 +70,45 @@ pub fn validate_coinbase(
 
         // validate BIP-141 segwit output
         if *tx.is_segwit {
-            let outputs = *tx.outputs;
+            let mut outputs = *tx.outputs;
             let mut is_wtxid_commitment_present: bool = false;
             let mut extracted_wtxid_commitment: ByteArray = "";
-            let mut i = 0;
-            let mut x = 6;
-
-            while i < outputs.len() {
-                let pk_script = *outputs[i].pk_script;
-
-                // check for OP_RETURN and the fixed prefix "0xaa21a9ed"
-                if pk_script.len() >= MINIMUM_WITNESS_COMMITMENT
-                    && pk_script[0] == 0x6a
-                    && pk_script[1] == 0x24
-                    && pk_script[2] == 0xaa
-                    && pk_script[3] == 0x21
-                    && pk_script[4] == 0xa9
-                    && pk_script[5] == 0xed {
+            let mut x = 0;
+
+            // construct expected wtxid commitment
+            let mut fixed_prefix_arr = array![0x6a, 0x24, 0xaa, 0x21, 0xa9, 0xed];
+            let mut fixed_prefix_byte: ByteArray = "";
```

Comment:
You can do `"\x6a\x24\xaa\x21\xa9\xed"`

Reply:
I tried this, but got this error " String literals can only include ASCII characters"

Reply:
Huh, strange, it works for "\x6a\x24"
Ok, then let's do the following:
1. Convert this to a felt (116705705699821)
2. Do `array.append_word(116705705699821, 6)`